unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/WebSocketCommunication.cs

```diff
@@ -8,24 +8,42 @@
 
 public class WebSocketCommunication : IKernelCommunication
 {
-    public static event Action<DCLWebSocketService> OnWebSocketServiceAdded;
     public static DCLWebSocketService service;
 
-    WebSocketServer ws;
-    private Coroutine updateCoroutine;
-    private bool requestStop = false;
+    [System.NonSerialized]
+    public static Queue<DCLWebSocketService.Message> queuedMessages = new Queue<DCLWebSocketService.Message>();
+
+    [System.NonSerialized]
+    public static volatile bool queuedMessagesDirty;
 
     private Dictionary<string, GameObject> bridgeGameObjects = new Dictionary<string, GameObject>();
 
     public Dictionary<string, string> messageTypeToBridgeName = new Dictionary<string, string>(); // Public to be able to modify it from `explorer-desktop`
+    private bool requestStop = false;
+    private Coroutine updateCoroutine;
 
-    [System.NonSerialized]
-    public static Queue<DCLWebSocketService.Message> queuedMessages = new Queue<DCLWebSocketService.Message>();
+    WebSocketServer ws;
 
-    [System.NonSerialized]
-    public static volatile bool queuedMessagesDirty;
+    public WebSocketCommunication(bool withSSL = false, int startPort = 7666, int endPort = 5100)
```

Comment:
@kuruk-mm , do I need to change also `endPort` here?

Reply:
Yes please. Try to use 7800 or 7766